jumpscripts/healthchecks/routeros_check.py

```diff
@@ -91,13 +94,27 @@ def checkros(c):
                     vfwid=vfwid, csname=c["name"], err=e
                 )
             )
-            return dict(
-                state="ERROR",
-                category=category,
-                message="RouterOS {roslink} on {spacelink} died".format(
-                    roslink=roslink, spacelink=spacelink
-                ),
-            )
+            message = "RouterOS {roslink} on {spacelink} died. Couldn't {action} RouterOS for restart"
```

Comment:
RouterOS --blabla-- on --blabla-- died. Couldn't start RouterOS for restart
This is what you'll get.
Nobody will understand this.
Make it as follows:
RouterOS --blabla-- on --blabla-- died. Tried to restart it. The restart failed in the stop action.

And similar for when the start fails off course